### PR TITLE
feat: add transformScale

### PR DESCRIPTION
**Pull request by xcqwan**
this adds support for parent DOM use transform scale
fix issues:
https://github.com/STRML/react-grid-layout/issues/484 
https://github.com/STRML/react-grid-layout/issues/350

### CODE REVIEW

**Comment by daynin** [comment]
@xcqwan thank you for your PR! Add documentation for this feature, pls

Let me know if you need my help

**Comment by xcqwan** [comment]
@daynin 
transformScale, the value of parent dom transform scale

example:
```
<div style={{ transform: 'scale(0.8)' }}>
    <ReactGridLayout
        {...otherProps}
        transformScale={0.8}
    />
</div>
```
or 
```
<div style={{ transform: 'scale(0.8)' }}>
    <div>
        <ReactGridLayout
            {...otherProps}
            transformScale={0.8}
        />
    </div>
</div>
```

**Comment by daynin** [comment]
@xcqwan thank you! But I mean we need to add some documentation in the readme file. We need it here because this is a change in public API

File: README.md [hunk 1]
```diff
@@ -301,6 +301,9 @@ isResizable: ?boolean = true,
 // Uses CSS3 translate() instead of position top/left.
 // This makes about 6x faster paint performance
 useCSSTransforms: ?boolean = true,
+// parent layout transform scale
+// If react-grid-layout dom's parent dom has transform scale, you should specify that key
```
[the remark this answers is not in the document]
Comment (xcqwan) [reply README.md]: @daynin you are right! I will correct. thanks

**Comment by daynin** [comment]
@xcqwan thank you for changing, but I think we still need some replacements here:

```
// parent layout transform scale
// if parent DOM node of ResponsiveReactGridLayout or ReactGridLayout has transform scale
// drag GridItem will get wrong position
// specify that key to fix it
transformScale: ?number = 1,
```
Let's write something like:
```
// If parent DOM node of ResponsiveReactGridLayout or ReactGridLayout has "transform: scale(n)" css property,
// we should set scale coefficient to avoid render artefacts while dragging.
transformScale: ?number = 1,
```

**Comment by daynin** [comment]
LGTM

@n1ghtmare what do you think?

**Comment by n1ghtmare** [comment]
@daynin LGTM as well (although I feel a bit uneasy adding more to the core API, however this seems to be a a common use case, I guess)

@xcqwan great job, thanks for contributing.

**Comment by xcqwan** [comment]
@n1ghtmare @daynin 
In the current situation, this is to solve the problem of simpler way.
The best solution， ReactGridLayout can automatic detection the 'transform: scale(n)' value

**Pull request merged**